docs/make.bat

```diff
@@ -46,15 +45,14 @@ if "%1" == "clean" (
 	for /d %%i in (%BUILDDIR%\*) do rmdir /q /s %%i
 	del /q /s %BUILDDIR%\*
 	for /d %%i in (source\*) do rmdir /q /s %%i
-
+	del /q /s source\extensions.rst
```

Comment:
No, it doesn't matter, in the sense that it'll still work fine regardless. But, it's just slightly less surprising/confusing to keep all the names consistent